src/types.js

```diff
@@ -155,6 +155,8 @@ export type Identity = $Diff<Auth, {| apiKey: string |}>;
 
 export type EmojiType = 'image' | 'unicode';
 
+export type LocalEmoji = {| emoji_name: string, emoji_code: string, emoji_type: EmojiType |};
```

Comment:
This type should get some jsdoc saying what it's for -- how it differs from the other object types we have for emoji.

In light of the comment above about naming, here's one version:
```suggestion
/** An emoji, in a form we can pass to `@zulip/shared`. */
export type EmojiForShared = {| emoji_name: string, emoji_code: string, emoji_type: EmojiType |};
```